portfolio/src/main/webapp/script.js

```diff
@@ -137,3 +137,29 @@ function createCommentElement(comment) {
 function deleteComments() {
   fetch('/delete-data', {method: 'POST'}).then(() => getComments());
 }
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+/* TODO (Carolyn): Update this chart to my actual bubble chart */
```

Comment:
Not a blocker, but an even better thing to do would be to open an issue in Github to track this change, then put the issue number here!